src/main/java/com/linkedin/kmf/services/configs/KafkaMetricsReporterServiceConfig.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.linkedin.kmf.services.configs;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+
+import java.util.Arrays;
+import java.util.Map;
+
+/**
+ * Created by IntelliJ IDEA.
+ * User: hackerwin7
+ * Date: 2018/01/18
+ * Time: 10:38 AM
+ * Desc:
+ */
+public class KafkaMetricsReporterServiceConfig extends AbstractConfig {
+
+  private static final ConfigDef CONFIG;
+
+  public static final String REPORT_METRICS_CONFIG = CommonServiceConfig.REPORT_METRICS_CONFIG;
+  public static final String REPORT_METRICS_DOC = CommonServiceConfig.REPORT_METRICS_DOC;
+
+  public static final String REPORT_INTERVAL_SEC_CONFIG = CommonServiceConfig.REPORT_INTERVAL_SEC_CONFIG;
+  public static final String REPORT_INTERVAL_SEC_DOC = CommonServiceConfig.REPORT_INTERVAL_SEC_DOC;
+
+  public static final String ZOOKEEPER_CONNECT_CONFIG = CommonServiceConfig.ZOOKEEPER_CONNECT_CONFIG;
+  public static final String ZOOKEEPER_CONNECT_DOC = CommonServiceConfig.ZOOKEEPER_CONNECT_DOC;
+
+  public static final String BOOTSTRAP_SERVERS_CONFIG = CommonServiceConfig.BOOTSTRAP_SERVERS_CONFIG;
+  public static final String BOOTSTRAP_SERVERS_DOC = CommonServiceConfig.BOOTSTRAP_SERVERS_DOC;
+
+  public static final String TOPIC_CONFIG = CommonServiceConfig.TOPIC_CONFIG;
+  public static final String TOPIC_DOC = CommonServiceConfig.TOPIC_DOC;
+
+  public static final String PRODUCER_ID_CONFIG = "produce.producer.id";
```

Comment:
It seems that Apache Kafka producer does not have this config. Could you tell me where is this config used?

Reply:
Seemed confused to me.  
`TOPIC_CONFIG` is for `KafkaMetricsReporterService` producer send topic.
`PRODUCER_ID_CONFIG` is for `KafkaMetricsReporterService` producer client id when constructed Apache Kafka producer.

Reply:
If the purpose of `produce.producer.id` is to be used as client.id of the producer, maybe you can just use the client.id directly? That can makes the config easier to understand.

BTW, is there any reason to explicitly specify client.id for producer here?

Reply:
Okay, maybe a fixed client.id suit for this.